### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wachsylon**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by wachsylon** [comment]
@conda-forge-admin, please rerender

**Comment by akrherz** [comment]
:wave: @wachsylon , can you describe the purpose of this PR some more?

**Comment by wachsylon** [comment]
@akrherz 
hi, the dev branch is used for cdo compilation with support for cmor. i wanted to update to a more recent version. i tried to test it locally with `conda-build`  but that took ages and then failed with os error `file too long` because the test environment is build sth like `placeholder/placeholder/placehoder`. 

Do you have an idea how to proceed here?

**Comment by akrherz** [comment]
the commit history with this PR is hard to follow.  Can the `dev` branch be rebased to `main` and then a smaller commit made to enable `cmor` ?

**Comment by wachsylon** [comment]
@akrherz sounds good to me. who should rebase? i will give it a try anyway

**Comment by wachsylon** [comment]
@akrherz can we somehow make this [branch](https://github.com/wachsylon/cdo-feedstock/tree/newdev) the new dev branch?

**Comment by akrherz** [comment]
@wachsylon , what's the commit hashes / 2 minor changes involved?

**Comment by wachsylon** [comment]
Bildschirmfoto vom 2023-07-20 10-49-20

**Comment by wachsylon** [comment]
@akrherz this is all it takes to run cdo with cmor i guess. i ran `conda-build` at my local pc which succeeded after hours of building the package.

**Comment by akrherz** [comment]
@wachsylon Thanks!  Is there a reason we don't just enable this on the `main` branch build of CDO?

**Comment by wachsylon** [comment]
reasons against it:

- cmor is a large package with a lot of dependencies which cause troubles from time to time
- I am the only developer of the linkage of cmor to cdo and only have limited time for debugging

**Comment by akrherz** [comment]
> reasons against it:

Thank you for educating me!  Let me see what can be done with the `dev` branch.

**Comment by akrherz** [comment]
@wachsylon lets continue this over on #142
